upgrade-tests/src/test/java/io/zeebe/test/RollingUpdateTest.java

```diff
@@ -119,7 +118,7 @@ public void shouldBeAbleToRestartContainerWithNewVersion() {
     // then
     try (final var client = newZeebeClient(containers.get(1))) {
       Awaitility.await()
-          .atMost(Duration.ofSeconds(5))
+          .atMost(Duration.ofSeconds(10))
```

Comment:
Did it become slower?

Reply:
Not exactly, it was always possibly flaky :) It means we wait up to 5 seconds for a broker to be detected as dead, but if you have a 1s probe interval, 2 required suspect probes, and a 2s probe timeout, it can take up to 5s to detect, so it was always going to be risky. I imagine if we looked around we'd find some flaky builds for this one